### ISSUE TITLE
mblevitus is broken

### ISSUE THREAD

**Issue by joa-quim** [dwcaress/MB-System#1098]
Using the example in man page
```
$ mblevitus -R-95/2 -V
...
 4000.0000    0.0000    0.0000   1894.4082
 4500.0000    0.0000    0.0000   1840.7629
 5000.0000    0.0000    0.0000   1742.9318
 5500.0000    0.0000    0.0000   1594.8563
 6000.0000    0.0000    0.0000   1390.4695
 6500.0000    0.0000    0.0000   1123.6957
 7000.0000    0.0000    0.0000    788.4505
 7500.0000    0.0000    0.0000    378.6410
 8000.0000    0.0000    0.0000   -111.8344
 8500.0000    0.0000    0.0000   -689.0861
 9000.0000    0.0000    0.0000   -1359.2330
 9500.0000    0.0000    0.0000   -2128.4031
10000.0000    0.0000    0.0000   -3002.7322
10500.0000    0.0000    0.0000   -3988.3657
11000.0000    0.0000    0.0000   -5091.4575
11500.0000    0.0000    0.0000   -6318.1699
12000.0000    0.0000    0.0000   -7674.6738
```
The break was in commit db7439d97b4cd2fc31966c719ddf90eae04f3976 when
```
pressure = 1.0052405 * depth[i] * (1.0 + 0.00528 * sin(DTR * latitude) * sin(DTR * latitude)) +
							  0.00000236 * depth[i] * depth[i];
			/* calculate water sound speed using DelGrosso equations */
			/* convert decibar to kg/cm**2 */
			pressure = pressure * 0.1019716;
```
was made into
```
      double pressure = 1.0052405 * depth[i] * (1.0 + 0.00528 * sin(DTR * latitude) * sin(DTR * latitude)) +
                 0.00000236 * depth[i] * depth[i]
        /* calculate water sound speed using
            DelGrosso equations */
        /* convert decibar to kg/cm**2 */
        * 0.1019716;
```
which is not the same equation.

**Comment by joa-quim** [comment]
BTW, shouldn't the equation use ``sin(DTR * lat_actual)`` instead of ``sin(DTR * latitude)``?

**Comment by dwcaress** [comment]
Thanks. The suggested fixes have been made and committed to master.

**Comment by dwcaress** [comment]
Fixed commit #1099
